package.json

```diff
@@ -52,7 +52,8 @@
     "ninja-util": "^1.3.1",
     "request": "^2.88.0",
     "request-promise-native": "^1.0.8",
-    "shortid": "^2.2.14"
+    "shortid": "^2.2.14",
+    "linked-list": "git+ssh://git@github.ibm.com:keg-core/linked-list.git"
```

Comment:
I believe this might be a problem. As this is a public library it should not refer no any private dependency.